src/include/86box/plat.h

```diff
@@ -125,6 +125,11 @@ extern void	plat_vid_reload_options(void);
 extern uint32_t plat_language_code(char* langcode);
 extern void plat_language_code_r(uint32_t lcid, char* outbuf, int len);
 
+extern void plat_clear_icon_set();
+extern void plat_system_icon_set();
+extern void plat_load_icon_set();
+extern void plat_get_icons_path(char* path_root);
+
```

Comment:
Why is this in the platform module? From what I see, icon sets are not used outside of the UI code, so this seems rather redundant to me.